scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer/objectdetector/SSDClassifierExample.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.mxnetexamples.infer.javapi.objectdetector;
+package org.apache.mxnetexamples.javaapi.infer.objectdetector;
```

Comment:
Why are we making this change? Curious if you just think that this structure is better or if something prompted it.

Reply:
You are right. In this way users can shop the collections of Java APIs instead of doing:
`mxnetexamples.benchmark.javaapi.*`
and 
`mxnetexmaples.infer.javaapi.*`

What do you think?

Reply:
Same apllies to `org.apache.mxnet.javaapi.infer.*`, so a single `.javaapi` would bring all from core, infer and examples

Reply:
Agree on this change after offline disucssion